packages/graphql/test/entities/Wizard.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { expect } from 'chai'
```

Comment:
Not sure if we want to test these in isolation or in a more e2e fashion by spawning a gql server. Either way, we have a navigation bug so it's good to add some tests to make sure it doesn't regress. Any recommendation for testing the gql logic @tgriesser?

Reply:
Yeah, I'd like advice on how to test this well.